### PR TITLE
Feature/cbi 1643 transaction fee

### PR DESCRIPTION
**Pull request by karthikeyan-cere**

### CODE REVIEW

File: backend/docker/sql/backend.sql [hunk 1]
```diff
@@ -384,3 +385,31 @@ CREATE TRIGGER transfer_count_trunc AFTER TRUNCATE ON extrinsic
 -- initialize the counter table
 UPDATE total SET count = (SELECT count(*) FROM extrinsic WHERE method IN ('transfer', 'transferKeepAlive')) WHERE name = 'transfers';
 COMMIT;
+
```
Comment (AndreiNavoichyk) [inline backend/docker/sql/backend.sql]: Remove it from backend.sql for now. We have it in migrations.

File: backend/docker/sql/backend-dev.sql [hunk 2]
```diff
@@ -384,3 +385,31 @@ CREATE TRIGGER transfer_count_trunc AFTER TRUNCATE ON extrinsic
 -- initialize the counter table
 UPDATE total SET count = (SELECT count(*) FROM extrinsic WHERE method IN ('transfer', 'transferKeepAlive')) WHERE name = 'transfers';
 COMMIT;
+
```
Comment (AndreiNavoichyk) [inline backend/docker/sql/backend-dev.sql]: Remove it from backend.sql for now. We have it in migrations.

File: frontend/components/Chain.vue [hunk 3]
```diff
@@ -172,6 +172,18 @@
           </div>
         </div>
       </div>
+      <div class="col-6 col-md-6 col-lg-3 mb-4">
+        <div class="card h-100">
+          <div class="card-body">
+            <h4 class="mb-3 title">
+              {{ $t('components.network.transactionFees') }}
```
Comment (AndreiNavoichyk) [inline frontend/components/Chain.vue]: Rename to `transaction**s**Fees`

File: backend/docker/sql/migration-2.sql [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+END;$$;
+CREATE CONSTRAINT TRIGGER transactions_fees_mod
+  AFTER INSERT OR DELETE ON extrinsic
+  DEFERRABLE INITIALLY DEFERRED
+  FOR EACH ROW EXECUTE PROCEDURE transactions_fees();
+-- TRUNCATE triggers must be FOR EACH STATEMENT
+CREATE TRIGGER transactions_fees_trunc AFTER TRUNCATE ON extrinsic
+  FOR EACH STATEMENT EXECUTE PROCEDURE transactions_fees();
+-- initialize the counter table
+UPDATE total SET count = COALESCE((SELECT SUM(CAST(fee_info::json->>'partialFee' AS INTEGER)) FROM extrinsic WHERE is_signed = 'true'),0) WHERE name = 'transactions_fees';
```
Comment (AndreiNavoichyk) [inline backend/docker/sql/migration-2.sql]: Check this query on DEV DB.

File: backend/docker/sql/backend.sql (hunk 1 above)
Comment (karthikeyan-cere) [reply backend/docker/sql/backend.sql]: Removed

File: backend/docker/sql/backend-dev.sql (hunk 2 above)
Comment (karthikeyan-cere) [reply backend/docker/sql/backend-dev.sql]: Removed

File: backend/docker/sql/migration-2.sql (hunk 4 above)
Comment (karthikeyan-cere) [reply backend/docker/sql/migration-2.sql]: Verified and updated.

File: frontend/components/Chain.vue (hunk 3 above)
Comment (karthikeyan-cere) [reply frontend/components/Chain.vue]: Updated

File: backend/docker/sql/migration-2.sql [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+END;$$;
+CREATE CONSTRAINT TRIGGER transactions_fees_mod
+  AFTER INSERT OR DELETE ON extrinsic
+  DEFERRABLE INITIALLY DEFERRED
+  FOR EACH ROW EXECUTE PROCEDURE transactions_fees();
+-- TRUNCATE triggers must be FOR EACH STATEMENT
+CREATE TRIGGER transactions_fees_trunc AFTER TRUNCATE ON extrinsic
+  FOR EACH STATEMENT EXECUTE PROCEDURE transactions_fees();
+-- initialize the counter table
+UPDATE total SET count = COALESCE((SELECT SUM(CAST(fee_info::json->>'partialFee' AS BIGINT)) FROM extrinsic WHERE is_signed = 'true' AND fee_info <> ''),0) WHERE name = 'transactions_fees';
```
Comment (AndreiNavoichyk) [inline backend/docker/sql/migration-2.sql]: Add space after `fee_info <>''),`

**Review by AndreiNavoichyk — APPROVED** [review]

File: backend/docker/sql/migration-2.sql (hunk 5 above)
Comment (karthikeyan-cere) [reply backend/docker/sql/migration-2.sql]: updated

**Pull request merged**